packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/resolution/ResolutionFeature.java

```diff
@@ -125,8 +125,7 @@ static Size computeBestPreviewSize(int cameraId, ResolutionPreset preset)
       }
     }
 
-    @SuppressWarnings("deprecation")
-    // TODO(camsim99): Suppression is currently safe because legacy code is used as a fallback for SDK >= S.
+    // TODO(camsim99): Suppression is currently safe because legacy code is used as a fallback for SDK < S.
```

Comment:
This was probably a typo? You have `if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {}` a few lines above.